docs/src/architecture/domain-model.md

```diff
@@ -15,13 +15,13 @@ Here _state_ refers to e.g. **Device Management** details as _installed software
 
   * **Measurement**:
      * has a _type name_
-     * carries a _series_ of **samples**
+     * carries a _sequence_ of **samples**
```

Comment:
The correct word is [time series](https://en.wikipedia.org/wiki/Time_series). A sequence is indexed by integers. A time-series by timestamps.

Reply:
Yes. The point is, I got the suggestion to avoid term "series" here, to prevent confusions with C8Y meaning of series. However I will reconsider that.

And finally, _time series of samples_ (by timestamps) is exactly what I meant here.

Reply:
Change to "time-series", see https://github.com/thin-edge/thin-edge.io/pull/1554/commits/bb58d5a1784b3b8e144704de8c2f41b5d71c9280